spec/presenters/email_alert_presenter_spec.rb

```diff
@@ -21,6 +38,8 @@
         email_alert_presenter_redrafted = EmailAlertPresenter.new(redrafted_cma_case)
         presented_data_redrafted = email_alert_presenter_redrafted.to_json
 
+        cma_links = DocumentLinksPresenter.new(cma_case).to_json[:links]
```

Comment:
Is related to the OPSS work or the other card?

Reply:
https://trello.com/c/zmaENgN4/881-pause-email-notification-for-cma-organisation-subscriptions-when-digital-markets-research-rerports-are-published

Reply:
It's related to both. I thought it was worth expanding this test (which just so happens to be using `cma_case` as its example) so that we check that the `links` hash is being correctly populated when we publish anything [expect `product_safety_alert_report_recall`](https://github.com/alphagov/specialist-publisher/blob/700bf3f6c4747e43e64acc77324c56f876751a95/app/presenters/email_alert_presenter.rb#L2) documents. The [new test](https://github.com/alphagov/specialist-publisher/blob/700bf3f6c4747e43e64acc77324c56f876751a95/spec/presenters/email_alert_presenter_spec.rb#L22) is checking that we send an empty links hash when we publish  product_safety_alert_report_recall doc types only. Make sense?

Reply:
Is it worth splitting the commit to have the new bits for CMA first? Not a blocker to merging.